app/src/main/res/values/strings.xml

```diff
@@ -438,6 +438,7 @@
     <string name="mute">Mute</string>
     <string name="unmute">Unmute</string>
     <string name="set_as_playlist_thumbnail">Set as playlist thumbnail</string>
+    <string name="unset_playlist_thumbnail">Unset thumbnail</string>
```

Comment:
"Unset playlist thumbnail" would probably be less confusing

Reply:
Soooorry, I mean "Unset permanent thumbnail"... I don't know why I wrote what I wrote :sweat_smile: :sweat_smile: